node-fs-writefile/cp.js

```diff
@@ -0,0 +1,15 @@
+import { readFile, writeFile } from 'node:fs/promises';
+async function copy() {
+  try {
+    const name = process.argv.slice(2);
+    const newFile = name.slice(1);
+    const fileTxt = await readFile(`${name}`, 'utf8');
+    const promise = await writeFile(`${newFile}`, `${fileTxt}`);
```

Comment:
`name` and `newFile` should be arrays at this point. I think you are just lucky this works.